src/components/Timeline.js

```diff
@@ -2,9 +2,35 @@ import React from 'react';
 import './Timeline.css';
 import TimelineEvent from './TimelineEvent';
 
-const Timeline = () => {
-  
-  return;
-}
+const Timeline = (props) => {
+
+  const timelineComponents = props.events.map((event, i) => {
+    return (
+      <p key={i}>
+        <TimelineEvent 
+          person={event.person} 
+          status={event.status} 
+          timeStamp={event.timeStamp}
+        />
+      </p>
```

Comment:
You can put the key into the TimelineEvent tag.

```suggestion
        <TimelineEvent 
          key={i}
          person={event.person} 
          status={event.status} 
          timeStamp={event.timeStamp}
        />
```